src/model/model.ts

```diff
@@ -216,12 +216,32 @@ export class Model {
     return false;
   }
 
+  // addPolicies adds policy rules to the model.
+  public addPolicies(sec: string, ptype: string, rules: string[][]): boolean {
+    const ast = this.model.get(sec)?.get(ptype);
+    if (!ast) {
+      return false;
+    }
+
+    for (const rule of rules) {
+      if (this.hasPolicy(sec, ptype, rule)) {
+        return false;
+      }
+    }
+
+    for (const rule of rules) {
```

Comment:
let's do `concat` here which maybe more efficient.

```js
ast.policy = ast.policy.concat(rules);
```

Reply:
Yeah, I will make this change